development/src/index.ts

```diff
@@ -334,6 +334,7 @@ const example = {
 					center: { lat: this.lat, lng: this.lng },
 					zoom: this.zoom,
 					clickableIcons: false,
+					mapId: process.env.GOOGLE_MAP_ID,
```

Comment:
You can load vector maps in Google Maps by providing your own map id... If left undefined it will default to the standard map.